infrastructure/main.tf

```diff
@@ -1,21 +1,29 @@
-provider "azurerm" {}
+provider "azurerm" {
+  version = "1.22.1"
+}
 
 locals {
-  ase_name            = "${data.terraform_remote_state.core_apps_compute.ase_name[0]}"
-  is_preview          = "${(var.env == "preview" || var.env == "spreview")}"
-  local_env           = "${local.is_preview ? "aat" : var.env}"
+  ase_name   = "${data.terraform_remote_state.core_apps_compute.ase_name[0]}"
+  is_preview = "${(var.env == "preview" || var.env == "spreview")}"
+  local_env  = "${local.is_preview ? "aat" : var.env}"
+  sku_size   = "${var.env == "prod" || var.env == "sprod" || var.env == "aat" ? "I2" : "I1"}"
```

Comment:
non of `s` prefixed environments are needed. previous version was good enough. `spreview` also not needed

Reply:
on the other hand, are those (`is_preview` and `local_env`) used anywhere?

Reply:
good catch removed them 😇